backend/utils.py

```diff
@@ -386,158 +388,207 @@ def find_route(tag1, tag2):
 
     return geom_route, route_dist, route
 
+
 def find_bicycle(tag1, tag2):
     ### Openrouteservie
-    api_key = os.environ.get('BICYCLE_API_KEY')
+    api_key = os.environ.get("BICYCLE_API_KEY")
     url = (
-    "https://api.openrouteservice.org/v2/directions/cycling-regular?api_key="
-    + api_key
-    + "&start="
-    + str(tag1[0])
-    + ","
-    + str(tag1[1])
-    + "&end="
-    + str(tag2[0])
-    + ","
-    + str(tag2[1])
-)
+        "https://api.openrouteservice.org/v2/directions/cycling-regular?api_key="
+        + api_key
+        + "&start="
+        + str(tag1[0])
+        + ","
+        + str(tag1[1])
+        + "&end="
+        + str(tag2[0])
+        + ","
+        + str(tag2[1])
+    )
     response = requests.get(url)
     if response.status_code == 200:
         geom = response.json()["features"][0]["geometry"]
-        geom_route = LineString(geom["coordinates"]).simplify(.05, preserve_topology=False)  # convert.decode_polyline(geom)
+        geom_route = LineString(geom["coordinates"]).simplify(
+            0.05, preserve_topology=False
+        )  # convert.decode_polyline(geom)
         route = True
-        route_dist = response.json()["features"][0]["properties"]["summary"]['distance'] / 1e3 #km
-        print('Bicycle length', round(route_dist, 1))
+        route_dist = (
+            response.json()["features"][0]["properties"]["summary"]["distance"] / 1e3
+        )  # km
+        print("Bicycle length", round(route_dist, 1))
     else:
         geom_route, route, route_idst = None, False, None
 
     return geom_route, route, route_dist
 
-#CREATE FERRY ROUTE
 
-def create_cost(world = train_intensity, buffer = 0 ):
-    '''
+# CREATE FERRY ROUTE
+
+
+def create_coast(world=train_intensity, buffer=0):
```

Comment:
:warning: This is the only functional change of this PR. I renamed the function from create_cost to create_coast and adapted usage accordingly